pulp_ansible/app/tasks/collections.py

```diff
@@ -103,7 +103,12 @@ def import_collection(artifact_pk, repository_pk=None):
             if collection_info[key] is None:
                 collection_info.pop(key)
 
-        collection_version = CollectionVersion(collection=collection, **collection_info)
+        collection_version = CollectionVersion(
+            collection=collection,
+            **collection_info,
+            contents=importer_result["contents"] or [],
```

Comment:
Instead of duplicated these defaults here is it possible the `galaxy_importer` could return the data and have pulp_ansible not interpret it?

Reply:
My goal is to avoid a situation where the default value `[]` is specified in multiple places inside pulp_ansible code.

Reply:
@bmbouter the next release of `galaxy-importer` sets these default values, I added these only to allow this to be merged beforehand and avoid null constraint errors.  I can remove this and we can wait to merge this until after `galaxy-importer` sets these values (will release after ansible 2.9 alpha available on pypi)